Applications/Resnet/jni/main.cpp

```diff
@@ -27,6 +27,7 @@
 #include <optimizer.h>
 
 #include <cifar_dataloader.h>
+#include <nntrainer_error.h>
```

Comment:
Examples like this resnet supposes to use ccapi which is open for the public. And unfortunately, nntrainer_error.h is not opened to the public and such defined macro such as ML_ERROR_NONE is not available here. We could consider if we can open nntrainer_error.h.. but for now, it might be better not to use them.

Reply:
thanks. it modified.